pkg/util/iptables/iptables.go

```diff
@@ -503,6 +518,18 @@ func getIPTablesWaitFlag(vstring string) []string {
 	}
 }
 
+// Checks if iptables-restore has a "wait" flag
+func getIPTablesRestoreWaitFlag(exec utilexec.Interface) restoreWaitFlag {
+	glog.V(4).Infof("checking if iptables-restore can wait")
+	bytes, err := exec.Command(cmdIPTablesRestore, "--help").CombinedOutput()
+	if err != nil {
+		return restoreWaitUnknown
+	} else if strings.Contains(string(bytes), "[-w secs]") {
```

Comment:
I can live with this in the mean time, since you will be the primary consumers of this for a while, it would be great to have a "clean" upstream, but I won't argue if you want to stick with this.